docs/api-reference.rst

```diff
@@ -9,3 +9,4 @@ API Reference
    api/searching
    api/model
    api/testing
+   api/maintenance
```

Comment:
I read the docs and I think it flows oddly now, with maintenance being the last section.  Could we move it up to just under "searching" instead?  That way it flows like: (1) general introduction to client; (2) docs about a particular thing you can do with client (searching), (3) docs about another thing you can do with client (manipulate maintenance mode).

Testing could also be considered a "thing you can do with client", but it's kind of a special case.